backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Services/UserService.cs

```diff
@@ -48,5 +50,24 @@ public async Task<AuthorizedParty> GetPartyFromReporteeListIfExists(int partyId)
             AuthorizedParty partyInfo = await _accessManagementClient.GetPartyFromReporteeListIfExists(partyId);
             return partyInfo;
         }
+
+        /// <inheritdoc/>
+        public async Task<List<RightHolder>> GetReporteeRightHolders(int partyId)
+        {
+            List<AuthorizedParty> rightHolders = await _accessManagementClient.GetReporteeRightHolders(partyId);
+
+            return rightHolders.Select(rightHolder => new RightHolder()
+            {
+                PartyUuid = rightHolder.PartyUuid,
+                PartyType = rightHolder.Type,
+                Name = rightHolder.Name,
+                PersonId = rightHolder.PersonId,
+                OrganizationNumber = rightHolder.OrganizationNumber,
+                UnitType = rightHolder.UnitType,
+                RegistryRoles = rightHolder.AuthorizedRoles.Where(role => Enum.IsDefined(typeof(RegistryRoleType), role.ToUpper()))
+                    .Select(role => (RegistryRoleType)Enum.Parse(typeof(RegistryRoleType), role.ToUpper()))
+                    .ToList()
```

Comment:
I noticed that the fake data can add the same role several times, can we add `.Distinct()` to this query to avoid duplicate roles?
```suggestion
                RegistryRoles = rightHolder.AuthorizedRoles.Where(role => Enum.IsDefined(typeof(RegistryRoleType), role.ToUpper()))
                    .Select(role => (RegistryRoleType)Enum.Parse(typeof(RegistryRoleType), role.ToUpper())).Distinct()
                    .ToList()
```

Reply:
Good point!
However, I think that is more a flaw of the automatically generated test data as no user will have multiples of the same role in prod (unless something is wrong)
How about I add the distinctiveness to the data generation instead?